template/lambda-sqs-worker-cdk/infra/__snapshots__/appStack.test.ts.snap

```diff
@@ -163,7 +163,7 @@ exports[`returns expected CloudFormation stack for dev 1`] = `
           },
         },
         "FunctionName": "serviceName",
-        "Handler": "app.handler",
+        "Handler": "index.handler",
```

Comment:
>The handler value will be automatically prefixed with the bundled output file name, index., unless the handler value contains a . character, in which case the handler value is used as-is to allow for values needed by some Lambda extensions.

Reply:
I wonder if we should rename the source file to `index.ts` to align 🤔

Reply:
I don't think it's very necessary, just a bundling thing tbh